tests/app/views/test_suppliers.py

```diff
@@ -1264,25 +1264,25 @@ def test_can_get_registered_frameworks_for_a_supplier(self):
             assert_equal(data['frameworks'], ['digital-outcomes-and-specialists'])
 
 
-class TestSupplierFrameworkUpdates(BaseApplicationTest, JSONUpdateTestMixin):
+class TestSupplierFrameworkResponse(BaseApplicationTest, JSONUpdateTestMixin):
     method = "post"
-    endpoint = "/suppliers/0/frameworks/digital-outcomes-and-specialists"
+    endpoint = "/suppliers/0/frameworks/g-cloud-6"
 
     def setup(self):
-        super(TestSupplierFrameworkUpdates, self).setup()
+        super(TestSupplierFrameworkResponse, self).setup()
 
         self.setup_dummy_suppliers(1)
         self.setup_dummy_user(1, role='supplier')
+        self.supplier_id = 0
+        self.framework_slug = 'g-cloud-6'
+        # can't do this unless the framework already exists, unfortunately
```

Comment:
Yeah, that's the idea. 

Can't get a `g-cloud-8` framework id because there's I guess the setup step for the class happens before the test fixture is execute (or at least that's my assumption).